sdk/rust/oak/src/grpc/server.rs

```diff
@@ -16,7 +16,7 @@
 
 //! Functionality to help Oak Nodes create gRPC pseudo-Nodes.
 
-use crate::{ReadHandle, OakStatus};
+use crate::{OakStatus, ReadHandle};
```

Comment:
Do you know why this was not formatted previously? Did the CI checks got removed accidentally?